addons/wowl/static/src/components/actionManager/actionManager.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <div t-name="wowl.ActionManager" owl="1">
+        <NavBar/>
```

Comment:
why the navbar inside the action manager?

Reply:
Most of its part are managed by the ActionManager: current menu (linked to the current action).
I am not sure of that design though...

Reply:
i disagree. the navbar is just connected to the menu service. that's it